android-agent/src/main/kotlin/io/opentelemetry/android/agent/session/SessionManager.kt

```diff
@@ -3,25 +3,21 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package io.opentelemetry.android.internal.session
+package io.opentelemetry.android.agent.session
 
-import io.opentelemetry.android.config.SessionConfig
-import io.opentelemetry.android.session.Session
-import io.opentelemetry.android.session.SessionIdGenerator
-import io.opentelemetry.android.session.SessionManager
-import io.opentelemetry.android.session.SessionObserver
-import io.opentelemetry.android.session.SessionStorage
+import io.opentelemetry.android.session.SessionProvider
 import io.opentelemetry.sdk.common.Clock
 import java.util.Collections.synchronizedList
 import kotlin.time.Duration
 
-internal class SessionManagerImpl(
+internal class SessionManager(
```

Comment:
The SessionManager will live only in the agent, and it will be our opinionated `SessionProvider` implementation.